package.json

```diff
@@ -40,7 +40,7 @@
     "eslint": "^8.27.0",
     "prettier": "^3.3.2",
     "release-it": "^15.5.0",
-    "typescript": ">=5.4.0"
+    "typescript": ">=5.6.0"
```

Comment:
Volar hugs TS pretty closely; this was necessary due to output lib code referencing SetIterator which is new to TS 5.6. Perhaps at some point we might need some upstream tweaks to make a little room for older Ember apps that are tied to older TS?